back-end/app/Repositories/Implementations/Ingredient.php

```diff
@@ -0,0 +1,25 @@
+<?php declare(strict_types=1);
+
+namespace App\Repositories\Implementations;
+
+use App\Models\Ingredient as IngredientModel;
+use App\Repositories\Interfaces\Ingredient as IngredientInt;
+
+/**
+ * Class Ingredient.
+ *
+ * The class specialize the common class method. For implement new feature
+ * add the firm inside the ingredient interface and implement here the method.
+ */
+class Ingredient extends Common implements IngredientInt
```

Comment:
La chiamerei proprio `IngredientRepository`. In questo modo evitiamo anche di fare aliasing.

Reply:
Mentre l'interfaccia o `IIngredientRepository` o anche direttamente con il namespace.